diffusers/long_prompt_weighting_stable_diffusion.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>The onnxruntime code snippet is missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>